src/leiningen/tach.clj

```diff
@@ -26,13 +26,24 @@
   [test-runner-main]
   (pr-str `(require '~test-runner-main)))
 
+(defn render-require-inject-ns
+  [planck?]
+  (pr-str `(do (require 'cljs.test)
```

Comment:
I think wrapping the `require`s in a `do` is breaking their ability to work. Perhaps instead conditionally a top level `(require 'cljs.test)` or `(require 'cljs.test 'planck.core)` would do it.